cdap-app-fabric/src/main/java/io/cdap/cdap/internal/tethering/TetheringAgentService.java

```diff
@@ -254,6 +254,8 @@ private void publishStartProgram(String runPayload, PeerInfo peerInfo) throws IO
     for (Map.Entry<String, String> cConfEntry : message.getCConfEntries().entrySet()) {
       cConfCopy.set(cConfEntry.getKey(), cConfEntry.getValue());
     }
+    // Don't upload event logs for tethered runs.
```

Comment:
would be good to mention why and link to the relevant jira

Reply:
Done